test/helper/enable.go

```diff
@@ -0,0 +1,11 @@
+package helper
+
+import (
+	"os"
+	"strings"
+)
+
+func Enabled(envvar string) bool {
+	value := strings.ToLower(os.Getenv(envvar))
+	return value == "1"
+}
```

Comment:
nit: I think this can be just checked if the ENV is set or not set

Reply:
I'd rather be more explicit here. What if you have `MYVAR=Off` or something, enabled would be confusing.

Still we can revisit once we do not need to run e2e testing by default for these kind of changes.